.github/workflows/docs.yaml

```diff
@@ -0,0 +1,14 @@
+name: Documentation Updates
+
+on: [push]
+
+jobs:
+    build-html:
+        name: Validate & Build HTML
+        runs-on: ubuntu-18.04
+        steps:
+            - uses: actions/checkout@v1
+            - uses: docker://cilium/docs-builder:latest
+              with:
+                entrypoint: ./Documentation/check-build.sh
+                args: html
```

Comment:
Will this work if external contributors push a change?

Reply:
I don't see a reason why not, GitHub actions that use the API are problematic with external folks, but this is just simple read-only job.